pyproject.toml

```diff
@@ -35,6 +35,7 @@ dynamic = ["version"]
 [project.optional-dependencies]
 dev = [
     "build",
+    "isort",
```

Comment:
The pinned deps are missing

Reply:
It was already added during the pylint PR and already exists

Reply:
ah I see, sorry!